src/main/java/org/apache/sysds/runtime/controlprogram/federated/FederatedData.java

```diff
@@ -105,6 +105,10 @@ public long getVarID() {
 		return _varID;
 	}
 
+	public void setFilepath(String filepath) {
+		_filepath = filepath;
+	}
+
```

Comment:
We might want to keep `FederatedData` constant and create a copy with a different filepath instead. I am open for discussion.